src/services/transactionService.ts

```diff
@@ -216,6 +224,9 @@ async function getTransactionDetailForTokenTransfer(
   if (!transaction) {
     return null;
   }
+  if (!receipt?.status) {
```

Comment:
So does receipt status is missing or the whole receipt doesn't get fetched?

Reply:
The `status` for failed transactions is `false` otherwise it's `true`